charts/consul/templates/api-gateway-controller-deployment.yaml

```diff
@@ -80,6 +85,9 @@ spec:
               -sds-server-host {{ template "consul.fullname" . }}-api-gateway-controller.{{ .Release.Namespace }}.svc \
               -k8s-namespace {{ .Release.Namespace }} \
               -log-level {{ default .Values.global.logLevel .Values.apiGateway.logLevel }} \
+              {{- if (and .Values.global.metrics.enabled .Values.global.metrics.enableGatewayMetrics) }}
+              -metrics-port 20200 \
```

Comment:
I think controller uses port `8080` by default. Would it make sense to use the default that way we don't need additional flag for this.

Reply:
Are you referring to the API Gateway controller or a different one? API Gateway defaults to a port of `0`/disabled if this flag isn't present.

Reply:
Keying off of the comment below about controller-runtime metrics, we'll likely, in the long run, want to see how simple it is to do something like register a pre-existing prometheus registry within a `github.com/armon/go-metrics` prometheus sink so we can use our standard library for metrics without requiring doing some scraping/aggregation.